module-sys/SystemManager/SystemManagerCommon.cpp

```diff
@@ -138,7 +138,7 @@ namespace sys
         case SystemManagerCommon::State::Running:
         case SystemManagerCommon::State::Suspend:
         case SystemManagerCommon::State::Shutdown:
-            exit(1);
+            abort();
```

Comment:
why is it abort now?

Reply:
After fixing of calling global destructors at exit(), each exit() call was finished with HF.
Previously I tried to remove all call to exit, but later decided to fix exit() no to clean up.

I will revert it back as at this moment it doesnt matter.